src/content/en/fundamentals/performance/webpack/decrease-frontend-size.md

```diff
@@ -78,7 +97,7 @@ function render(data, target) {
 
 <li>
 
-The UglifyJS plugin minifies it into approximately the following:
+A minificator compresses it into approximately the following:
```

Comment:
A minifier? (I haven't come across the term minificator before)

Reply:
Yep, you’re right, thanks :–)